src/openrct2/peep/PeepAnimationData.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include "../entity/Peep.h"
+
+#include <span>
+
+namespace OpenRCT2
+{
+    struct SpriteBounds
+    {
+        uint8_t sprite_width;           // 0x00
```

Comment:
Is this a new struct? If so, please use camelCase. If not, you can leave it as-is.

Reply:
It was moved from the old `PeepData.h`, indeed. I could change the code style, but that would make the diff noisier with irrelevant changes. Better to leave that for now, I think.

Reply:
Yeah, it’s why I asked if it was a new struct.